server/konnectors/currencies.coffee

```diff
@@ -0,0 +1,152 @@
+americano = require 'americano-cozy'
+request = require 'request'
+moment = require 'moment'
+crypto = require 'crypto'
+async = require 'async'
```

Comment:
In the repository root directory, run:

    npm install xml2js --save 

It will add the dependency to your local installation and add a new line to the `package.json` file to include it in every installation.